cheatsheets/Secure_Product_Design_Cheat_sheet.md

```diff
@@ -0,0 +1,92 @@
+
+Separation of duties is a fundamental principle of internal control in business and organizations. It is a system of checks and balances that ensures that no single individual has control over all aspects of a transaction. This is done by assigning different tasks to different people, so that no one person has control over the entire process. This helps to reduce the risk of fraud and errors, as well as ensuring that all tasks are completed in a timely manner. Separation of duties is an important part of any organization's internal control system, and is essential for maintaining the integrity of the organization's financial records.
+
+### 2. The principle of Defence in Depth
+
+The principle of Defence in Depth is a security strategy that involves multiple layers of security controls to protect an organization’s assets. It is based on the idea that if one layer of security fails, the other layers will still be able to protect the asset. The layers of security can include physical security, network security, application security, and data security. The goal of Defence in Depth is to create a secure environment that is resilient to attack and can quickly detect and respond to any security incidents. By implementing multiple layers of security, organizations can reduce the risk of a successful attack and minimize the damage caused by any successful attack.
+
+### 3. The principal of Zero Trust
+
+Zero Trust is a security model that assumes that all users, devices, and networks are untrusted and must be verified before access is granted. It is based on the idea that organizations should not trust any user, device, or network, even if they are inside the organization’s network. Instead, all requests for access must be authenticated and authorized before access is granted. Zero Trust also requires organizations to continuously monitor and audit user activity to ensure that access is only granted to those who need it. This model is designed to reduce the risk of data breaches and other security incidents by ensuring that only authorized users have access to sensitive data.
+
+### 4. The principal of Security in the Open
+
+Security in the Open is a concept that emphasizes the importance of security in open source software development. It focuses on the need for developers to be aware of the security implications of their code and to take steps to ensure that their code is secure. This includes using secure coding practices, testing for vulnerabilities, and using secure development tools. Security in the Open also encourages developers to collaborate with security experts to ensure that their code is secure. Finally, it emphasizes the need for developers to be aware of the security implications of their code and to take steps to ensure that their code is secure.
```

Comment:
Seems somewhat repetitive:
"It focuses on the need for developers to be aware of the security implications of their code and to take steps to ensure that their code is secure"
"Finally, it emphasizes the need for developers to be aware of the security implications of their code and to take steps to ensure that their code is secure."

Reply:
Probably needs hyphenated as well: "Security-in-the-Open".

Reply:
Removed the repeated line (flow of thought and self editing FTW). Added the hyphenation.